contrib/win32/openssh/install-sshd.ps1

```diff
@@ -274,13 +274,25 @@ sc.exe privs sshd SeAssignPrimaryTokenPrivilege
 Add-Privilege -Account $sshdSid -Privilege SeAssignPrimaryTokenPrivilege
 Add-Privilege -Account $sshdSid -Privilege SeServiceLogonRight
 
+# create logs folder and set its permissions
 if(-not (test-path $logsdir -PathType Container))
 {
     $null = New-Item $logsdir -ItemType Directory -Force -ErrorAction Stop
 }
-$rights = [System.Security.AccessControl.FileSystemRights]"Read, Write"
-$accessRule = New-Object System.Security.AccessControl.FileSystemAccessRule($sshdAccount, $rights, "ContainerInherit,ObjectInherit", "None", "Allow")
 $acl = Get-Acl -Path $logsdir
-$Acl.SetAccessRule($accessRule)
+$acl.SetSecurityDescriptorSddlForm("O:BAG:DUD:PAI(A;OICI;FA;;;SY)(A;OICI;FA;;;BA)")
```

Comment:
can you please add some comments what does this mean?
How does this impact the sftp-server.log?